From e6c373b026e80369360ec0e379033bc0a312f7f7 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Tue, 2 Jan 2018 14:05:35 +0200
Subject: [PATCH 29/57] cl-som-imx8: dts: Update the platform DTS's

1) Fix the copyrigh string;
2) Remove unused stuff;
3) hdmi-audio set correct protocol value.

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm64/boot/dts/compulab/cl-som-imx8-4k.dts |  9 +---
 arch/arm64/boot/dts/compulab/cl-som-imx8.dts    | 57 +------------------------
 2 files changed, 3 insertions(+), 63 deletions(-)

diff --git a/arch/arm64/boot/dts/compulab/cl-som-imx8-4k.dts b/arch/arm64/boot/dts/compulab/cl-som-imx8-4k.dts
index 1984325..6855733 100644
--- a/arch/arm64/boot/dts/compulab/cl-som-imx8-4k.dts
+++ b/arch/arm64/boot/dts/compulab/cl-som-imx8-4k.dts
@@ -1,6 +1,5 @@
 /*
- * Copyright (C) 2016 Freescale Semiconductor, Inc.
- * Copyright 2017 NXP
+ * Copyright (C) 2017 CopuLab Ltd.
  *
  * This program is free software; you can redistribute it and/or
  * modify it under the terms of the GNU General Public License
@@ -15,12 +14,6 @@
 
 #include "cl-som-imx8.dts"
 
-/ {
-	sound-hdmi {
-		video-mode = <95>;
-	};
-};
-
 &dcss {
 	disp-mode = <95>;	/* #97: 3840x2160p@30Hz 16:9 */
 };
diff --git a/arch/arm64/boot/dts/compulab/cl-som-imx8.dts b/arch/arm64/boot/dts/compulab/cl-som-imx8.dts
index b629000..886de9b 100644
--- a/arch/arm64/boot/dts/compulab/cl-som-imx8.dts
+++ b/arch/arm64/boot/dts/compulab/cl-som-imx8.dts
@@ -1,6 +1,5 @@
 /*
- * Copyright (C) 2016 Freescale Semiconductor, Inc.
- * Copyright 2017 NXP
+ * Copyright (C) 2017 CopuLab Ltd.
  *
  * This program is free software; you can redistribute it and/or
  * modify it under the terms of the GNU General Public License
@@ -95,7 +94,7 @@
 		compatible = "fsl,imx-audio-cdnhdmi";
 		model = "imx-audio-hdmi";
 		audio-cpu = <&sai4>;
-		protocol = <1>;
+		protocol = <0>;
 	};
 
 	sound-spdif {
@@ -380,11 +379,6 @@
 	status = "okay";
 };
 
-&hdmi {
-	disp-dev = "hdmi_disp";
-	status = "okay";
-};
-
 &i2c2 {
 	clock-frequency = <100000>;
 	pinctrl-names = "default";
@@ -508,7 +502,6 @@
 };
 #endif
 
-#if 1
 &pcie1{
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_pcie1>;
@@ -519,7 +512,6 @@
 	hard-wired = <1>;
 	status = "okay";
 };
-#endif
 
 &pwm2 {
 	pinctrl-names = "default";
@@ -553,43 +545,6 @@
 	status = "okay";
 };
 
-&hdmi_cec {
-	status = "okay";
-};
-
-#if 0
-&lcdif {
-	status = "disabled";
-	disp-dev = "mipi_dsi_northwest";
-	display = <&display0>;
-
-	display0: display@0 {
-		bits-per-pixel = <24>;
-		bus-width = <24>;
-
-		display-timings {
-			native-mode = <&timing0>;
-			timing0: timing0 {
-			clock-frequency = <9200000>;
-			hactive = <480>;
-			vactive = <272>;
-			hfront-porch = <8>;
-			hback-porch = <4>;
-			hsync-len = <41>;
-			vback-porch = <2>;
-			vfront-porch = <4>;
-			vsync-len = <10>;
-
-			hsync-active = <0>;
-			vsync-active = <0>;
-			de-active = <1>;
-			pixelclk-active = <0>;
-			};
-		};
-	};
-};
-#endif
-
 &usdhc1 {
 	pinctrl-names = "default", "state_100mhz", "state_200mhz";
 	pinctrl-0 = <&pinctrl_usdhc1>;
@@ -668,14 +623,6 @@
 	status = "okay";
 };
 
-&spdif2 {
-	assigned-clocks = <&clk IMX8MQ_CLK_SPDIF2_SRC>,
-			<&clk IMX8MQ_CLK_SPDIF2_DIV>;
-	assigned-clock-parents = <&clk IMX8MQ_AUDIO_PLL1_OUT>;
-	assigned-clock-rates = <0>, <24576000>;
-	status = "okay";
-};
-
 &gpu_pd {
 	power-supply = <&sw1a_reg>;
 };
-- 
1.9.1

